### PR TITLE
Fixes #41 update with real runner output

### LINKED ISSUE #41: Update with generated output from QualificationRunner

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: examples/minimal/input/Itraconazole/Itraconazole.json [hunk 1]
```diff
@@ -972,7 +972,7 @@
       }
     },
     {
-      "Name": "Midazolan 600 mg SD",
```
Comment (msevestre) [inline examples/minimal/input/Itraconazole/Itraconazole.json]: This was a typo here!

**Comment by msevestre** [comment]
I would not know why xlsx would not be compatible with linux ?!?
But yes I can also export as csv. I think we should  absolutely export to xlsx too so that we can quickly on windows check that everything is as expected
Let's make it another issue so that we can work with the updated value here for Pierre

**Comment by msevestre** [comment]
@Yuri05 Results are exported as csv. Where do you see XLSX?
image

**Comment by Yuri05** [comment]
>Results are exported as csv. Where do you see XLSX?

ok, my fault, is CSV indeed. merging

**Pull request merged**